msal/tests/dev apps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -47,13 +47,14 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
             LoadApplication(new App());
 
+            // To activate system webview, remove '//' from line 51 below, and comment out line 57 ;
             App.UIParent = new UIParent(this);
 
             #region Web browsers for MSAL.NET Android
-            // To activate embedded webview, remove '//' from line 56 below, 
-            // and comment out line 50 above -> App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
+            // To activate embedded webview, remove '//' from line 57 below, 
+            // and comment out line 51 above -> App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
 
-            //App.UIParent = new UIParent(this, true);
+            App.UIParent = new UIParent(this, true);
```

Comment:
>App.UIParent = new UIParent(this, true); [](start = 12, length = 40)

you now have both UIParent setters enabled.... really should rather be a meaningful if statement or as mentioned... a setting in the UI.

Reply:
this one is still set 2 times.... please see above

---
In reply to: [223935008](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r223935008) [](ancestors = 223935008)